opendevin/runtime/client/runtime.py

```diff
@@ -286,75 +280,3 @@ def get_working_directory(self):
     # Overwrite the init_sandbox_plugins
     def init_sandbox_plugins(self, plugins: list[PluginRequirement]) -> None:
         pass
-
-
-async def test_run_command():
-    sid = 'test'
-    cli_session = 'main' + ('_' + sid if sid else '')
-    event_stream = EventStream(cli_session)
-    runtime = EventStreamRuntime(
-        sandbox_config=config.sandbox, event_stream=event_stream, sid=sid
-    )
-    await runtime.ainit()
-    await runtime.run_action(CmdRunAction('ls -l'))
-
-
-async def test_event_stream():
```

Comment:
Can we bring this function back and add `config = load_app_config()` on the first line, so we have some "demo" to run?

Reply:
I can do that, but it's not our standard practice to put this sort of stuff in a main function in library files, usually we put it in the "tests" directory. Should we move it there?

Reply:
Yeah agree -  maybe this goes better into tests, we can probably move it there.

Reply:
Or i can do this in a follow up PR since you already revive it back #3128

Reply:
OK, let's resolve this in a future PR. Thanks for reviewing!

Reply:
Actually, the revival of the test was causing a circular import, so I just deleted it for now. Let's move it to the tests directory in the future PR.